### ISSUE TITLE
Incorrect password error is not accessible

### ISSUE THREAD

**Issue by willeybryan** [cds-snc/notification-api#248]
**Steps to Reproduce**
- Sign into service with an incorrect email and/or password
- https://notification.alpha.canada.ca/sign-in
- Read error message and look at the forgot password link in the error message

**Expected**
- The entire error message itself should be in Red text to identify itself as an error
- Forgot password link is already at the bottom, don't duplicate it in the error message

**Actual**
- The forgot password link in the error message is Red

**Notes**
- Red should be reserved for the error itself not links. Can cause users to miss.
- From UX audit Slide 13

![image](https://user-images.githubusercontent.com/46505127/63040480-87d71300-be93-11e9-8bb8-e855981a1bc4.png)
